### DISCUSSION TITLE
cuDNN 9.0.0?

### DISCUSSION THREAD

**Discussion by ghost** [Purfview/whisper-standalone-win#187] [Q&A] [answered]
lets try this.https://developer.nvidia.com/cudnn-downloads?target_os=Windows&target_arch=x86_64&target_version=Agnostic&cuda_version=11

**Comment by Purfview** [comment, marked as answer]
Current CTranslate2 is using cuDNN v8.x API, most likely that 9.x is incompatible.
In 9.0.0 I see "flash attention" improvements, but flash attention is not implemented in current CTranslate2.

Anyway, you can try if this hack works:
Delete `cudnn_cnn_infer64_8.dll` and `cudnn_ops_infer64_8.dll` and copy 9.0.0 files from there -> https://we.tl/t-DQEcvONhpW

**Reply by Purfview** [reply]
@toprak did you tested it?

**Reply by ghost** [reply]
I mean i can test it but dont have time to do such a detailed comparison like before

**Reply by Purfview** [reply]
just test if it works at all

**Reply by ghost** [reply]
nope did not work Detecting language using up to the first 30 seconds. Use `--language` to specify the language
Could not load library cudnn_ops_infer64_8.dll. Error code 126
Please make sure cudnn_ops_infer64_8.dll is in your library path!

**Reply by Purfview** [reply]
Ok, I got idea for another hack.
